packages/eslint-config-expo/utils/react.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     'react/no-unknown-property': 'warn',
     'react/require-render-return': 'warn',
     'react/react-in-jsx-scope': 'off',
+    'react/no-unescaped-entities': 'off',
```

Comment:
Slightly out of scope, but I responded on your task about the React plugin linter rules that I think will address this in the future.

Reply:
Merged the React rules PR so this should be addressed https://github.com/expo/expo/pull/28032